build/build/src/engine/context.rs

```diff
@@ -651,7 +651,7 @@ pub async fn runner_sanity_test(
     );
     if enso_java.is_none() {
         let test_base = Command::new(&repo_root.runner)
-            .args(["--run", repo_root.test.join("Base_Tests").as_str(), "^Text"])
+            .args(["--run", repo_root.test.join("Base_Tests").as_str()])
```

Comment:
Since we can now run `enso --run test/Base_Test`, why don't we get rid of the factorial_enso code above? It seems no longer necessary.

Reply:
Removed in 562e611f6b